bpfd/src/server/bpf.rs

```diff
@@ -117,13 +134,18 @@ impl<'a> BpfManager<'a> {
                 current_position: None,
                 metadata: Metadata {
                     priority,
+                    proceed_on_mask,
```

Comment:
Including this in the `Metadata` will affect the sort.
When priority, name, and attached are equal, the `proceed_on_mask` will be used as a tie-breaker.
This might not be intended.